src/style.css

```diff
@@ -1 +1,81 @@
-/* Styles go here */
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  font-size: 0;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  text-align: center;
+  align-items: center;
+}
+
+.nav__list {
+  padding: 0;
+}
+
+.nav {
+  background-color: #fff;
+  height: 60px;
+  margin-right: 50px;
+}
+
+.img {
+  margin-left: 50px;
+}
+
+.nav__item {
+  display: inline-block;
+  text-decoration: none;
+  color: black;
+  margin-left: 20px;
+}
+
+.nav__link {
+  font-size: 12px;
+  line-height: 60px;
+  text-transform: uppercase;
+  text-decoration: none;
+  padding: 23px 0;
+  position: relative;
+  color: black;
+}
+
+.nav__link:visited {
+  color: black;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.nav__link:focus {
+  color: #00acdc;
+}
```

Comment:
Optional:

It is a great practice to create variables for the colors/fonts you use in your code. 
Imagine the page was redesigned - what will you do? You will need to fix all the colors/fonts manually. But if you use a variable - you can just change it in a single place